specs/Badge.md

```diff
@@ -0,0 +1,106 @@
+# Badge
```

Comment:
Add a link to the Epic issue for Badge Convergence

Reply:
Added https://github.com/microsoft/fluentui/issues/16254 is it correct?

Reply:
#16925 is the issue to link, but it looks like you did that.